client/browser/src/shared/code-hosts/gerrit/codeHost.module.scss

```
@@ -0,0 +1,49 @@
+.hover-overlay--gerrit {
+    background-color: #ffffff;
+    border: 1px solid rgba(0, 0, 0, 0.125);
+    box-shadow: 0 1px 5px rgba(0, 0, 0, 0.3);
+}
+
+/* Styles for the hover overlay */
+.hover-overlay__contents code {
```

Let's convert classes from BEM notation to concise names because they are scoped automatically by CSS modules. Applies to other classes with `__` in the selector name.

```suggestion
.contents code {
```